### PR TITLE
Added Outline VPN tutorial

### PR DESCRIPTION
**Pull request by ntimo**
I wrote an article on how to Setup [Outline VPN](https://getoutline.org/)

I have read and understood the Contributor's Certificate of Origin
available at the end of https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Timo 0mfhniozkb9s4q7e6ap8yvlt@nowitzki.me

### CODE REVIEW

File: tutorials/vpn-server-with-outline/01.en.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+SPDX-License-Identifier: MIT
+path: "/tutorials/vpn-server-with-outline"
+slug: "vpn-server-with-outline"
+date: "2019-03-15"
+title: "VPN server with Outline"
+short_description: "This tutorial will walk you trough the process on how to setup outline"
+tags: ["Outline", "VPN", "Security"]
+author: "ntimo"
+author_img: "https://avatars3.githubusercontent.com/u/6145026"
+---
```
Comment (n-se) [inline tutorials/vpn-server-with-outline/01.en.md]: Please do not remove variables you don't wanna fill. You keep them empty, like `""`.

File: tutorials/vpn-server-with-outline/01.en.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+* Choose the resources you need from Type
+* Click on the field `User Data` enter this `#include https://get.docker.com` (this will install docker)
+* Select your SSH-Key 
+    * You can read [this](https://help.github.com/en/enterprise/2.16/user/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) article to know how to generate SSH key
+* Write your server hostname in `name` input
+* Click `Create & Buy Now`
+
+## Step 2 - Download the clients
+
+You will need to download two clients. The Outline Manager to administrate your Server and the Outline client to establish the VPN connection.
```
Comment (n-se) [inline tutorials/vpn-server-with-outline/01.en.md]:
```suggestion
You will need to download two clients. The Outline Manager to administrate your server and the Outline client to establish the VPN connection.
```

File: tutorials/vpn-server-with-outline/01.en.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+## Step 3 - Install updates on the server (optional)
+
+Wait a few seconds for the server to startup and then connect to it using SSH and your private key.
+Now you should install available updates on your server by running:  
+
+`apt update && apt upgrade -y`  
+
+## Step 4 - Install the Outline Server
+
+Once you have installed the Outline manager and the outline client you can go ahead and open the manager. Once it has started select `Setup Outline anywhere`.
```
Comment (n-se) [inline tutorials/vpn-server-with-outline/01.en.md]:
```suggestion
Once you have installed the Outline manager and the Outline client you can go ahead and open the manager. Once it has started select `Setup Outline anywhere`.  
```

File: tutorials/vpn-server-with-outline/01.en.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+Wait a few seconds for the server to startup and then connect to it using SSH and your private key.
+Now you should install available updates on your server by running:  
+
+`apt update && apt upgrade -y`  
+
+## Step 4 - Install the Outline Server
+
+Once you have installed the Outline manager and the outline client you can go ahead and open the manager. Once it has started select `Setup Outline anywhere`.  
+
+Now we are going to install the server it self. Do that simply connect via SSH to your server and execute the following command:
```
Comment (n-se) [inline tutorials/vpn-server-with-outline/01.en.md]:
```suggestion
Now we are going to install the server it self. To do that execute the following command:  
```
You are already connected via SSH, if you follow the tutorial step by step this is maybe confusing.

File: tutorials/vpn-server-with-outline/01.en.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+`apt update && apt upgrade -y`  
+
+## Step 4 - Install the Outline Server
+
+Once you have installed the Outline manager and the outline client you can go ahead and open the manager. Once it has started select `Setup Outline anywhere`.  
+
+Now we are going to install the server it self. Do that simply connect via SSH to your server and execute the following command:  
+```
+sudo bash -c "$(wget -qO- https://raw.githubusercontent.com/Jigsaw-Code/outline-server/master/src/server_manager/install_scripts/install_server.sh)"
```
Comment (n-se) [inline tutorials/vpn-server-with-outline/01.en.md]:
```suggestion
curl -sSL https://raw.githubusercontent.com/Jigsaw-Code/outline-server/master/src/server_manager/install_scripts/install_server.sh | bash
```
You don't have "sudo" by default on Debian. If you mean the initial solution is better, just close this suggestion.

File: tutorials/vpn-server-with-outline/01.en.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+```
+
+When you are asked if you want to install Docker answear with `y`.
```
Comment (n-se) [inline tutorials/vpn-server-with-outline/01.en.md]:
```suggestion
When you are asked if you want to install Docker answer with `y`.  
```

File: tutorials/vpn-server-with-outline/01.en.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+If you have connection problems, it may be that your router or cloud provider
+blocks inbound connections, even though your machine seems to allow them.
+
+Make sure to open the following ports on your firewall, router or cloud provider:
+- Management port 6311, for TCP
+- Access key port 48682, for TCP and UDP
+```
+
+Copy the following output from the ssh console:
```
Comment (n-se) [inline tutorials/vpn-server-with-outline/01.en.md]:
```suggestion
Copy the following output from the SSH console:  
```

File: tutorials/vpn-server-with-outline/01.en.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+Make sure to open the following ports on your firewall, router or cloud provider:
+- Management port 6311, for TCP
+- Access key port 48682, for TCP and UDP
+```
+
+Copy the following output from the ssh console:  
+`{"apiUrl":"https://10.0.0.1:6311/dPvv3xoyNNzEpR9rC_RKQA" "certSha256":"C517734428F9AC58D37373E81466930799C13C770C1809B8CECAAB30B06F0F42"}`  
+And paste it into the Outline manager. Once you click on `Done` you will be presented with the connections dashboard.  
+
+## Step 5 - Connect to your vpn server
```
Comment (n-se) [inline tutorials/vpn-server-with-outline/01.en.md]:
```suggestion
## Step 5 - Connect to your VPN server
```

File: tutorials/vpn-server-with-outline/01.en.md [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+Once you have installed the Outline manager and the outline client you can go ahead and open the manager. Once it has started select `Setup Outline anywhere`.  
+
+Now we are going to install the server it self. Do that simply connect via SSH to your server and execute the following command:  
+```
+sudo bash -c "$(wget -qO- https://raw.githubusercontent.com/Jigsaw-Code/outline-server/master/src/server_manager/install_scripts/install_server.sh)"
+```
+
+When you are asked if you want to install Docker answear with `y`.  
+
+Now you should get a output in your log that looks like this:
```
Comment (n-se) [inline tutorials/vpn-server-with-outline/01.en.md]:
```suggestion
Now you should get an output in your log that looks like this:  
```

File: tutorials/vpn-server-with-outline/01.en.md [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+
+## Step 5 - Connect to your vpn server
+
+In the manager you can click on the computer icon near the `My access key`. Now you can select if you want to connect this device or another device.  
+If you choose this device the Outline client will start and connect to your server.  
+If you choose `connect a different device` you will get a s3 link that you can simply copy and open on the other device. You will then be shown the ssh:// link that you will need to enter in the Outline client on that device.
+
+## Step 6 -  Enable automatic server upgrades
+
+If you don't want to have to worry about your Server being up to date, you can simply enable automatic upgrades by running:
```
Comment (n-se) [inline tutorials/vpn-server-with-outline/01.en.md]:
```suggestion
If you don't want to have to worry about your server being up to date, you can simply enable automatic upgrades by running:  
```

File: tutorials/vpn-server-with-outline/01.en.md [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+## Step 5 - Connect to your vpn server
+
+In the manager you can click on the computer icon near the `My access key`. Now you can select if you want to connect this device or another device.  
+If you choose this device the Outline client will start and connect to your server.  
+If you choose `connect a different device` you will get a s3 link that you can simply copy and open on the other device. You will then be shown the ssh:// link that you will need to enter in the Outline client on that device.
+
+## Step 6 -  Enable automatic server upgrades
+
+If you don't want to have to worry about your Server being up to date, you can simply enable automatic upgrades by running:  
+`sudo dpkg-reconfigure -plow unattended-upgrades`
```
Comment (n-se) [inline tutorials/vpn-server-with-outline/01.en.md]:
```suggestion
`dpkg-reconfigure -plow unattended-upgrades`  
```
At the beginning we logged in as root user.

**Comment by ntimo** [comment]
Thank you @n-se for the review :)

**Pull request merged**